### PR TITLE
53861: Backend service creation: add get demandes

### PR DESCRIPTION
**Pull request by mkrout**

### CODE REVIEW

**Review by hakermi — APPROVED** [review]
LGTM

File: processes-api/src/main/java/org/exoplatform/processes/model/Demande.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.Set;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Demande {
```
Comment (plamarque) [inline processes-api/src/main/java/org/exoplatform/processes/model/Demande.java:30]: @hakermi @mkrout  english please > Request

**Review by plamarque — COMMENTED** [review]
Class names, tables, endpoint uris and named queries should be in english. 

Demandes may be literally translated as Requests. In more traditional workflow lingua franca we use the term Workflow and WorkflowDefinition (DemandeType)

Please revert and refactor.

**Comment by mkrout** [comment]
OK, Will change demande to request and demande type to workflow

**Comment by plamarque** [comment]
better change Demande to Workflow and DemandeType to WorkflowDefinition.

**Comment by plamarque** [comment]
or maybe simpler : DemandeType > Workflow
Demande > Work

**Comment by plamarque** [comment]
see here for a full glossary with synonyms : http://www.aiai.ed.ac.uk/project/wfmc/ARCHIVE/DOCS/glossary/glossary.html

Let’s avoid collision with other concepts in eXo (such as activities ans tasks)

**Comment by mkrout** [comment]
Ok

**Pull request merged**